Soccar [Game-Server]/src/nl/soccar/gameserver/util/RoomUtilities.java

```diff
@@ -15,6 +17,12 @@
     private RoomUtilities() {
     }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)